.gitignore

```diff
@@ -45,3 +45,6 @@ venv/
 # e2e
 /ui/e2e/*.js
 /ui/e2e/*.map
+
+# OS generated files
+.DS_Store
```

Comment:
tiny nit: newline so we don't see the ugly :no_entry_sign:

Reply:
Thanks, will do! Don't like the 🚫  either :(